documentation/2.0/content/userguide/tools/compare.md

```diff
@@ -285,10 +285,18 @@ Comparing the new and old models:
 
  To compare the two model files and generate the output to files, run the tool as follows:
 
-    $ weblogic-deploy\bin\compareModel.cmd -oracle_home c:\wls12213 -output_dir c:\cm-output ew_model.yaml old_model.yaml
+    $ weblogic-deploy\bin\compareModel.cmd -oracle_home c:\wls12213 -output_dir c:\cm-output new_model.yaml old_model.yaml
 
     The following files will be written to the directory:
 
     diffed_model.json
     diffed_model.yaml
     compare_model_stdout
+
+## Parameter table for compareModel.
```

Comment:
Can I suggest that you downsize this heading a bit by using ### or #### (to be in line with the other headings on this page) and removing the period at the end of the heading; it's not necessary.